### PR TITLE
Rewrite Go driver for latest bblfsh SDK version

### LINKED ISSUE #3: Rewrite Go driver

### PR DESCRIPTION
**Pull request by dennwc**
Resolves #3

Signed-off-by: Denys Smirnov <denys@sourced.tech>

### CODE REVIEW

**Comment by dennwc** [comment]
Still missing UAST transformers and annotations. Also needs testing.

File: native/native_test.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"gopkg.in/bblfsh/sdk.v1/protocol"
+	"gopkg.in/bblfsh/sdk.v1/sdk/driver"
+)
+
+func TestNative(t *testing.T) {
+	const dir = "../fixtures"
```
Comment (juanjux) [inline native/native_test.go]: This is a little redundant considering that the integration tests will also check those files, maybe checking a single string with code and that the native response is Ok is enough here.

**Comment by juanjux** [comment]
Other than the CI failing, looks good to me, it can be merged if you fix the CI and generate the integration tests (`bblfsh-sdk-tools fixtures --language=go fixtures/*.go` once you've installed the driver in the server)so we've a version with the native parse working and then you can work on the annotations on another PR.

**Comment by dennwc** [comment]
Requires https://github.com/bblfsh/sdk/pull/232 (and SDK release tag) to pass CI.

**Comment by juanjux** [comment]
Merged the PR and released SDK's 1.12.0, it's still failing, looks like you need to update the .travis.yml to the latest version (bblfsh-sdk update).

**Comment by juanjux** [comment]
SDK PR merged and tagged, it seem there are still some failing tests.

**Comment by dennwc** [comment]
@juanjux PTAL

**Pull request merged**